indico/modules/events/client/js/util/list_generator.js

```diff
@@ -39,10 +39,11 @@
       $(this)
         .closest('tr')
         .toggleClass('selected', this.checked);
-      $('.js-requires-selected-row').toggleClass(
-        'disabled',
-        !$('.list input:checkbox:checked').length
-      );
+      const rows = $('.list input:checkbox:checked').length;
+      $('.js-requires-selected-row').toggleClass('disabled', !rows);
+      if (rows > 1) {
+        $('.js-requires-selected-row.single-row').toggleClass('disabled', true);
```

Comment:
Wouldn't be more intuitive (w/o the if)?

```js
$('.js-requires-selected-row.single-row').toggleClass('disabled', rows != 1);
```

Reply:
Sure